web/turkey/docs/build/html/_static/pygments.css

```diff
@@ -0,0 +1,65 @@
+.highlight .hll { background-color: #ffffcc }
```

Comment:
Not sure been a long time since I worked with it. There are online guides for this though.